### PR TITLE
Create Giardini.biz.xml

### PR DESCRIPTION
**Pull request by ryanbr**

### CODE REVIEW

File: src/chrome/content/rules/Giardini.biz.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!--
+	Refused:
+		giardini.biz (non-www)
```
Comment (J0WI) [inline src/chrome/content/rules/Giardini.biz.xml]: Can be redirected to www.

File: src/chrome/content/rules/Giardini.biz.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!--
+	
+	Refused:
+		css.giardini.biz
+		js.giardini.biz
+		old.giardini.biz
+-->
+<ruleset name="giardini.biz">
```
Comment (J0WI) [inline src/chrome/content/rules/Giardini.biz.xml]:
```suggestion
<ruleset name="giardini.biz">
	<target host="giardini.biz" />
```

File: src/chrome/content/rules/Giardini.biz.xml (hunk 1 above)
Comment (J0WI) [reply src/chrome/content/rules/Giardini.biz.xml]: Oops, that host does not even exist.

Comment (J0WI) [reply src/chrome/content/rules/Giardini.biz.xml]: > Oops, that host does not even exist.

@ryanbr could you remove all references to the non-existent host?

**Comment by zoracon** [comment]
```
ERROR src/chrome/content/rules/Giardini.biz.xml: Fetch error: http://giardini.biz/ => https://www.giardini.biz/: (6, 'Could not resolve host: giardini.biz')
```

Is this host resolving for you @J0WI?

**Comment by pipboy96** [comment]
@zoracon `giardini.biz` is NOERROR but it's empty

**Comment by J0WI** [comment]
```
$ curl http://giardini.biz/
curl: (6) Could not resolve host: giardini.biz
```

**Comment by pipboy96** [comment]
@J0WI The DNS record is empty, there are no `A` records.

**Pull request merged**